package.json

```diff
@@ -55,7 +55,7 @@
 		"yup": "^0.32.11"
 	},
 	"peerDependencies": {
-		"@lifesg/react-design-system": "^2.3.0-canary.7",
+		"@lifesg/react-design-system": "^2.4.0-canary.4",
```

Comment:
can trouble you to reflect the latest version in the `README.md` too?

Reply:
Updated, thanks for remind